Patches/Forsakens/Forsakens_CE_Patch_ThingDefs_Weapons_BlackHydra.xml

```diff
@@ -0,0 +1,117 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Patch>
+	<Operation Class="PatchOperationFindMod">
+		<mods>
+			<li>Forsakens</li>
+		</mods>
+		<match Class="PatchOperationSequence">
+			<operations>
+			
+				<!-- NOTE: Black Hydra is currently disabled for CE, as the original mod incorrectly uses a thingDef rather than ThingDef, which is incompatible with CE patching -->
```

Comment:
It can, of course, be patched manually, you know.

Reply:
I've tried using PatchOps to manually remove then recreate (add) the Black Hydra back in as a CE weapon, except that they throw a continuous stream of errors in the debug log regarding invalid weapon for hunting jobs (among other problems).

However, as soon as I edited the original mod to replace `thingDef` with `ThingDef`, the current patches work perfectly without errors - pawns can hunt using this weapon (although this is not advisable anyway), melee tools work properly and projectiles are CE compatible (save for replacing a custom Forsakens class with CE's own projectile one).

Mod author @Kik0hi has also agreed (via Steam Workshop) that the `thingDef` vs `ThingDef` issue will be fixed on his end in a future update:

![Annotation 2020-04-10 215848](https://user-images.githubusercontent.com/4993380/78996118-7eaf5a80-7b76-11ea-9bd3-277803f175b4.png)

For now, I'm perfectly happy to temporarily dummy out this weapon, as its absence doesn't adversely affect faction pawn generation and the general gameplay experience.

Reply:
Fair enough.